src/components/app/data/hooks/useEnterpriseCourseEnrollments.js

```diff
@@ -140,21 +147,21 @@ export default function useEnterpriseCourseEnrollments(queryOptions = {}) {
 
   // TODO: Talk about how we don't have access to weeksToComplete on the dashboard page.
   const allEnrollmentsByStatus = useMemo(() => transformAllEnrollmentsByStatus({
-    enterpriseCourseEnrollments,
+    enrollmentsByStatus: enterpriseCourseEnrollments.enrollmentsByStatus,
     requests,
     contentAssignments,
-  }), [contentAssignments, enterpriseCourseEnrollments, requests]);
+  }), [contentAssignments, enterpriseCourseEnrollments.enrollmentsByStatus, requests]);
 
   return useMemo(() => ({
     data: {
       allEnrollmentsByStatus,
-      enterpriseCourseEnrollments,
+      enterpriseCourseEnrollments: enterpriseCourseEnrollments.enrollments,
       contentAssignments,
       requests,
     },
   }), [
     allEnrollmentsByStatus,
-    enterpriseCourseEnrollments,
+    enterpriseCourseEnrollments.enrollments,
```

Comment:
[nit] should this be 
`enterpriseCourseEnrollments: enterpriseCourseEnrollments.enrollments,`

Reply:
This dependency array is an array of values, not an object of key/value pairs 😄